docker-compose.yml

```
@@ -22,13 +22,18 @@ services:
       - pgdata:/var/lib/postgresql/data
   pgadmin:
     environment:
-      PGADMIN_DEFAULT_EMAIL: test@user
+      PGADMIN_DEFAULT_EMAIL: test@user.com
```

`test@user` is a valid email address, any reason to add `.com`? 

```suggestion
      PGADMIN_DEFAULT_EMAIL: test@user
```

---

I'll revert this so it doesn't block the other changes